docs/stamping.md

```diff
@@ -15,20 +15,6 @@ argument, as well as `BUILD_HOST` and `BUILD_USER`. You can supply more with the
 
 Some rules accept an attribute that uses the status variables.
 
-## Substitutions attribute
-
-In a `pkg_npm` you can use the `substitutions` attribute like:
-
-```python
-pkg_npm(
-    name = "npm_package",
-    substitutions = {"0.0.0-PLACEHOLDER": "{STABLE_GIT_COMMIT}"},
```

Comment:
With this rule+attribute gone should we just delete the file? Is stamping.md even referenced anywhere? (I can't find anything)